1907-05-27.xml

```diff
@@ -1503,6 +1499,256 @@
             </div>
                    
             <div type="page" n="2"> </div>
```

Comment:
the `</div>` tag on this line closes page two (which you've just started!). It needs to go at the end of all of the contents.